src/main/java/org/veupathdb/service/eda/data/plugin/selfcorrelation/SelfCorrelationUnipartitenetworkPlugin.java

```diff
@@ -66,10 +66,16 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
     ByteArrayInputStream statsIn = new ByteArrayInputStream(statsBytes.toByteArray());
     dataStreams.put("statsFile.json", statsIn);
 
+    // some default filtering thresholds
+    Number correlationCoefThreshold = getPluginSpec().getCorrelationCoefThreshold() != null ? getPluginSpec().getCorrelationCoefThreshold() : 0.2;
+    Number pValueThreshold = getPluginSpec().getSignificanceThreshold() != null ? getPluginSpec().getSignificanceThreshold() : 0.05;
+
     useRConnectionWithRemoteFiles(Resources.RSERVE_URL, dataStreams, connection -> {
       connection.voidEval("corrResult <- jsonlite::read_json('statsFile.json', simplifyVector = TRUE)");
       connection.voidEval("edgeList <- corrResult$statistics");
       connection.voidEval("names(edgeList) <- c('source', 'target', 'weight', 'pValue')");
+      connection.voidEval("edgeList <- edgeList[edgeList$pValue <= " + pValueThreshold + "]");
+      connection.voidEval("edgeList <- edgeList[abs(edgeList$weight) >= " + correlationCoefThreshold + "]");
```

Comment:
it's been a bit, but i think we're missing a comma after the thresholds

Reply:
im gonna blame ai lol